test/msg.go

```diff
@@ -64,31 +65,39 @@ func (m *mockMsg) FlowUUID() string {
 	return m.flow.UUID
 }
 
-func (m *mockMsg) Channel() courier.Channel      { return m.channel }
-func (m *mockMsg) ID() courier.MsgID             { return m.id }
-func (m *mockMsg) EventID() int64                { return int64(m.id) }
-func (m *mockMsg) UUID() courier.MsgUUID         { return m.uuid }
-func (m *mockMsg) Text() string                  { return m.text }
-func (m *mockMsg) Attachments() []string         { return m.attachments }
-func (m *mockMsg) Locale() i18n.Locale           { return m.locale }
-func (m *mockMsg) ExternalID() string            { return m.externalID }
-func (m *mockMsg) URN() urns.URN                 { return m.urn }
-func (m *mockMsg) URNAuth() string               { return m.urnAuth }
-func (m *mockMsg) ContactName() string           { return m.contactName }
-func (m *mockMsg) HighPriority() bool            { return m.highPriority }
-func (m *mockMsg) QuickReplies() []string        { return m.quickReplies }
-func (m *mockMsg) Origin() courier.MsgOrigin     { return m.origin }
-func (m *mockMsg) ContactLastSeenOn() *time.Time { return m.contactLastSeenOn }
-func (m *mockMsg) Topic() string                 { return m.topic }
-func (m *mockMsg) ResponseToExternalID() string  { return m.responseToExternalID }
-func (m *mockMsg) Metadata() json.RawMessage     { return m.metadata }
-func (m *mockMsg) IsResend() bool                { return m.isResend }
-func (m *mockMsg) ReceivedOn() *time.Time        { return m.receivedOn }
-func (m *mockMsg) SentOn() *time.Time            { return m.sentOn }
-func (m *mockMsg) WiredOn() *time.Time           { return m.wiredOn }
+func (m *mockMsg) Channel() courier.Channel         { return m.channel }
+func (m *mockMsg) ID() courier.MsgID                { return m.id }
+func (m *mockMsg) EventID() int64                   { return int64(m.id) }
+func (m *mockMsg) UUID() courier.MsgUUID            { return m.uuid }
+func (m *mockMsg) Text() string                     { return m.text }
+func (m *mockMsg) Attachments() []string            { return m.attachments }
+func (m *mockMsg) Locale() i18n.Locale              { return m.locale }
+func (m *mockMsg) ExternalID() string               { return m.externalID }
+func (m *mockMsg) URN() urns.URN                    { return m.urn }
+func (m *mockMsg) URNAuth() string                  { return m.urnAuth }
+func (m *mockMsg) URNAuthTokens() map[string]string { return m.urnAuthTokens }
```

Comment:
this is bit confusing but I think we should move methods that are only used for testing elsewhere